share/dbtypes_gen.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
For safety, we should add a `from __futur__ import print_function` here as the first import line.